Documentation/template/index.html.js

```diff
@@ -1,18 +1,15 @@
 function transform(model, _attrs){
```

Comment:
I am thinking using underscore prefix to stand for a `docfx` generated property..